lib/ReactViews/Workbench/Controls/ShortReport.tsx

```diff
@@ -39,45 +45,72 @@ export default class ShortReport extends React.Component<{
   }
 
   render() {
+    const shortReportSections = this.props.item?.shortReportSections?.filter(
+      r => isDefined(r.name)
+    );
+
     if (
       (!isDefined(this.props.item.shortReport) ||
         this.props.item.shortReport === "") &&
-      (!isDefined(this.props.item.shortReportSections) ||
-        this.props.item.shortReportSections.length === 0)
+      (!isDefined(shortReportSections) || shortReportSections.length === 0)
     ) {
       return null;
     }
+
     return (
-      <div className={Styles.shortReport}>
-        {isDefined(this.props.item.shortReport)
-          ? parseCustomMarkdownToReact(this.props.item.shortReport, {
+      <Box fullWidth displayInlineBlock padded>
+        {/* Show shortReport */}
+        {isDefined(this.props.item.shortReport) && (
+          <Text textLight medium>
+            {parseCustomMarkdownToReact(this.props.item.shortReport, {
               catalogItem: this.props.item
-            })
-          : ""}
-        {this.props.item.shortReportSections
-          .filter(r => isDefined(r.name))
-          .map((r, i) => (
-            <div key={r.name}>
-              <a
-                href="#"
-                onClick={this.clickShortReport.bind(this, r.name)}
-                className={Styles.shortReportTitle}
-              >
+            })}
+          </Text>
+        )}
+
+        {/* Show shortReportSections */}
+        {shortReportSections.map((r, i) => (
+          <React.Fragment key={r.name}>
+            <RawButton
+              fullWidth
+              onClick={this.clickShortReport.bind(this, r.name)}
+              css={`
+                text-align: left;
+              `}
+              aria-expanded={r.show}
+              aria-controls={`${this.props.item.uniqueId}-${r.name}`}
+            >
+              <Text as="span" textLight bold medium>
                 {r.name}
-                {r.show ? (
-                  <Icon glyph={Icon.GLYPHS.minusThick} />
-                ) : (
-                  <Icon glyph={Icon.GLYPHS.plusThick} />
-                )}
-              </a>
-              {r.show && isDefined(r.content)
-                ? parseCustomMarkdownToReact(r.content, {
-                    catalogItem: this.props.item
-                  })
-                : ""}
-            </div>
-          ))}
-      </div>
+              </Text>
+
+              <StyledIcon
+                styledWidth={"8px"}
+                light
+                glyph={r.show ? GLYPHS.minusThick : GLYPHS.plusThick}
+                css={`
+                  padding-left: 10px;
+                  display: inline;
```

Comment:
Don't be afraid to update the styled component props themselves if it makes sense to, e.g. a `StyledIcon` can feasibly often be inline